<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigureGraphicsAdvanced</class>
 <widget class="QWidget" name="ConfigureGraphicsAdvanced">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>404</width>
    <height>376</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="accessibleName">
   <string>Advanced</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_1">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QGroupBox" name="groupBox_1">
       <property name="title">
        <string>Advanced Graphics Settings</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QWidget" name="populate_target" native="true">
          <layout class="QVBoxLayout" name="verticalLayout">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_VertexClamping">
             <item>
              <widget class="QLabel" name="label_VertexClamping">
               <property name="text">
                <string>Vertex Clamping:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="vertex_clamping"/>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QCheckBox" name="recompress_astc_textures">
             <property name="text">
              <string>Recompress ASTC Textures (Improves Stability)</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_ShaderAccuracy">
             <item>
              <widget class="QLabel" name="label_ShaderAccuracy">
               <property name="text">
                <string>Shader Accuracy:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="shader_accuracy_mode_combobox"/>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QCheckBox" name="enable_nvidia_byte_swap_workaround">
             <property name="toolTip">
              <string>This is a workaround for a specific NVIDIA driver bug seen in Super Mario RPG. It may or may not improve stability or performance in other titles.</string>
             </property>
             <property name="text">
              <string>Enable NVIDIA Byte Swap Workaround (Super Mario RPG)</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="opengl_disable_fast_buffer_sub_data">
             <property name="toolTip">
              <string>May help with buffer update issues on some OpenGL drivers, potentially at a performance cost. Default is unchecked (Fast path enabled).</string>
             </property>
             <property name="text">
              <string>Disable Fast Buffer Sub-Data (OpenGL only)</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
